test/studies/isx/.gitignore

```diff
@@ -0,0 +1 @@
+*.good
```

Comment:
Oh, really?  That seems unfortunately centralized.  Is there a reason that that's better?